com.valvesoftware.Steam.yml

```diff
@@ -180,13 +180,12 @@ modules:
         sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop
         sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop
         desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
-        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
-        desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
         sed -i s:/usr/lib/:/app/lib/: steam
         PREFIX=/app make install
         ln -sf /bin/true /app/bin/steamdeps
         cp /usr/bin/cmp /app/bin
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
+        ln -sf /app/bin/steam-wrapper /app/bin/steam
```

Comment:
I feel that `/app/bin/steam` originally being a symlink to `/app/lib/steam/bin_steam.sh` is Steam's own implementation detail and can change at any time. For example, it could become a script or even a binary file in any update.

With that in mind, it's safer to move `/app/bin/steam` someplace else instead of assuming that it's a symlink that we can remove.

Reply:
I don't think adding symlink buys anything. We could do test -x /app/lib/steam/bin_steam.sh to see the file exists in manifest so it fails in build time if this changes. Note that this path points to /app, it cannot arbitrarily change by without app rebuild.

Reply:
Also, the path was suggested by @smcv, I am assuming they wouldn't have suggested it if it was likely to change.